src/vm/gcenv.ee.cpp

```diff
@@ -1192,7 +1192,7 @@ namespace
     bool CreateSuspendableThread(
         void (*threadStart)(void*),
         void* argument,
-        const char* name)
+        const wchar_t* name)
```

Comment:
Changes in this file are about hoisting the conversion from char to wchar into the 'CreateThread' api in the interface (and then all helpers take wchar.   This allows the conversion code to be done once (and shared).